doc/api/vm.md

```diff
@@ -898,6 +920,54 @@ to disallow any changes to it.
 Corresponds to the `[[RequestedModules]]` field of [Cyclic Module Record][]s in
 the ECMAScript specification.
 
+### `sourceTextModule.instantiate()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {undefined}
+
+Instantiate the module with the linked requested modules.
+
+This resolves the imported bindings of the module, including re-exported
+binding names. When there are any bindings that cannot be resolved,
+an error would be thrown synchronously.
+
+If the requested modules include cyclic dependencies, the
+[`sourceTextModule.linkRequests(modules)`][] method must be called on all
+modules in the cycle before calling this method.
+
+### `sourceTextModule.linkRequests(modules)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `modules` {vm.Module\[]} Array of `vm.Module` objects that this module depends on.
+  The order of the modules in the array is the order of
+  [`sourceTextModule.moduleRequests`][].
+* Returns: {undefined}
+
+Link module dependencies. This method must be called before evaluation, and
+can only be called once per module.
+
+The order of the module instances in the `modules` array should correspond to the order of
+[`sourceTextModule.moduleRequests`][] being resolved.
```

Comment:
```suggestion
[`sourceTextModule.moduleRequests`][] being resolved. If two module requests have the same specifier and
import attributes, they must be resolved the same module instance or an `ERR_MODULE_LINK_MISMATCH`
would be thrown. For example, when linking requests for this module:

```mjs
import foo from 'foo';
import { bar } from 'foo';
```
The array must contain two references to the same instance, because the two module requests are identical.

```